### PR TITLE
AWS auto case - byo IAM profile 417

### PR DESCRIPTION
**Pull request by yunjiang29**
backport of https://github.com/openshift/release/pull/58256

### CODE REVIEW

**Comment by yunjiang29** [comment]
@gpei PTAL

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml [hunk 1]
```diff
@@ -56,6 +56,36 @@ resources:
       cpu: 400m
       memory: 800Mi
 tests:
+- as: aws-ipi-byo-iam-profile-master-f28
```
Comment (gpei) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml:59]: @yunjiang29 seems there's no `arm` keyword in this two job names, do we want to add it?

Comment (yunjiang29) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml]: Updated.

**Comment by gpei** [comment]
cc @liangxia for the approval, thanks

**Pull request merged**